app/src/main/res/layout/ongoing_story_card.xml

```diff
@@ -29,9 +29,10 @@
       <ImageView
         android:id="@+id/lesson_thumbnail"
         android:layout_width="80dp"
-        android:layout_height="44dp"
+        android:layout_height="0dp"
         android:layout_marginStart="8dp"
         android:importantForAccessibility="no"
+        app:layout_constraintDimensionRatio="16:9"
         android:paddingStart="40dp"
```

Comment:
Remove these paddings and update scale type to `android:scaleType="centerInside"`

Reply:
I have removed extra paddings from right and left. and changed the `scaleType` to `centerInside`.

Reply:
@Luffy18346 remove padding top too.

Reply:
@rt4914, but after removing padding top, the image will not look same as in the mock I have checked.

Reply:
That is a really good point but actually currently app contains dummy images and in real images will contain padding on its own therefore we need to remove this padding.

Reply:
Removed padding top and finally pushed the code.

Reply:
Looks good.